Add linters

AJ193 opened the pull request:
I have add 
- linters
- README file

Removed 
- murple logo
- app screenshot
- MIT.md file

AJ193 committed:
Add linters

i have add linters,README file,Removed murple logo,app screenshot,MIT.md file

AJ193 committed:
Installation of npm linter

I have installed things
lighthouse
webhint
stylelint

pinkmoon25 commented on README.md:
```diff
@@ -1,2 +1,97 @@
+An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more.
+
+You can get the Lighthouse report by any of the following ways:
+
```
- [x] Good job setting up the project, however please use the correct readme file, currently you have added the linters config readme, The readme file is the documentation of our projects, It contains details about our project like the project title, Key features, a description and so on, please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Copy the Readme.md file from the above link and use it as a template, fill in the details to customise it to your projec such as the description, tech stack , title, and so on, add a MIT license file to your project's root folder and linkit to the readme, remove the microverse logo too :+1: 
here is a pic of the readme template :point_down: 
![image](https://user-images.githubusercontent.com/97521557/211616860-0fd6e5ab-383d-4a07-97b2-fd725b740cdb.png)

pinkmoon25 reviewed (changes requested):
Hi @AJ193 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media2.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif"/>

## Highlights
linter setup :heavy_check_mark: 
github flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AJ193 committed:
Edit the Readme

In this Change i editied the README.md file.And also added a MIT.md file

brainconnect93 commented on README.md:
```diff
@@ -1,288 +1,36 @@
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
```
- [ ] As requested by the previous reviewer kindly add the table of content and the other aspect of the README file back to the README file `i.e` project title, Key features, a description, the future features add 2-3 thing you will like to include in the project in the future

brainconnect93 reviewed (changes requested):
### Hi @AJ193 , 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media.tenor.com/isiLRnxz3zwAAAAM/cat-driving-serious.gif" width="500px" height="300px">

### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AJ193 commented:
@brainconnect93 I need help to complete this project please help me.

AJ193 committed:
Edit MIT.md

In this change I Edited Readme.md file in which i added Table of contents, etc.
And also i edited MIT.md file.

Owusu-Desmond commented on README.md:
```diff
@@ -1,36 +1,68 @@
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
```
- [x] Please since you are the only contributor to this project you can singularize the `Authors` section to `Author` and replace `Author1` with your name. :smile: I also notice this author section is missing this back to top `<p align="right">(<a href="#readme-top">back to top</a>)</p>` which is very important.

Owusu-Desmond commented on README.md:
````diff
@@ -1,36 +1,68 @@
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
````
- [x] Good job :+1: but as the previous reviewer requested, Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections are present in yours. But now comparing your readme to this [Microverse Readme template](https://github.com/microverseinc/readme-template), you will see that you are missing some necessary headings like `key features`, `Future Features`, and ` Getting Started` that describe how a new developer could make use of your project by following some steps.

Owusu-Desmond reviewed (changes requested):
Hi @AJ193  :wave: 

You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.

<img src="https://media1.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif"/>

 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AJ193 committed:
Edit Readme.md file

In this Change i editied the Readme.md file.

AJ193 committed:
Edit the Readme.md file

In this i editied the readme.md file.

jerryowusu commented on README.md:
```diff
@@ -1,288 +1,72 @@
 
-> Describe 1 - 3 features you will add to the project.
+- GitHub: [@AJ193](https://github.com/AJ193)
+- Twitter: [@GhulamShabbir59](https://twitter.com/GhulamShabbir59)
+- LinkedIn: [ghulamshabbir225264247](www.linkedin.com/in/ghulam-shabbir-225264247)
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- CONTRIBUTING -->
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🤝 Contributing <a name="contributing"></a>
 
```
As requested by the previous reviewer, your readme is missing certain key/important sections of the [Microverse Readme template](https://github.com/microverseinc/readme-template) like the `Key Features`, `Future Features`, `Getting Started` etc. Note that it is a requirement to produce all sections of the readme template in this and all other subsequent projects you do in Microverse here. Kindly add them and make sure to customized to match your project credentials. :pray:

AJ193 replied:
@jerryowusu I know i removed them because i don't have any future features that are going to be added to this document. Should i include them and leave them empty?

AJ193 replied:
@brainconnect93 @Owusu-Desmond @PinkMoon25 I know i removed them because i don't have any future features that are going to be added to this document. Should i include them and leave them empty?
I only have 2 extra reviews left so please guide me so that I can complete this project and not waste my reviews.

jerryowusu reviewed (changes requested):
Hi @AJ193 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so-close](https://user-images.githubusercontent.com/65233860/211763317-9eed1100-a7c7-4fd2-b7e1-5d24df5457e1.gif)

 ## Highlights
All linters are passing :heavy_check_mark: 
Great effort so far :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Consider updating your pull request title to a more suitable one that defines what your whole project is about since the current one `Add linters` does not tell the whole picture :+1: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AJ193 committed:
Edit Readme.md file

AJ193 committed:
Update to the Readme.md file

AJ193 committed:
Update MIT.md

AJ193 committed:
modify readme.md file

AJ193 committed:
Merge branch 'Add-test.md/file' of https://github.com/AJ193/Hello-Microverse into Add-test.md/file

bimbolabuari reviewed (commented):
Hello @AJ193  👋,

_This is an additional review as a result of the second opinion request._

- Kudos on making a great effort on your first project in Microverse 👍 
  - The previously requested changes are important parts of the README file, however, since this is your first project in Microverse we can overlook it, moving on, you will need to add all the README requirements in your future projects 🙂 
 

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Welcome to Microverse and keep rocking 🚀!

huseyinbiyik reviewed (approved):
Hi @AJ193,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Alt Text](https://media.giphy.com/media/wZjlCH43M3M0U/giphy.gif)



## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝

## Optional suggestions
- [OPTIONAL]You can change this template text with your project name.
<img width="877" alt="Screenshot 2023-01-12 at 10 24 16 PM" src="https://user-images.githubusercontent.com/25887253/212162186-6cf90f4b-72c2-4648-af52-4327175de3ee.png">


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
